### ISSUE TITLE
ctculture - cleanup request

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#8066]
@AJLinn here's the code table you requested.

We are supporting markdown in description now, feel free to use that.

Let me know in https://github.com/ArctosDB/dev/issues/42 if you'd like to modernize the structure before tackling anything.

You might consider losing the commas in values, they seem to find ways to break things (mostly list operations).

Talk to me before going too crazy, but as long as you leave the ``culture`` column alone we should be able to use it to get back to Arctos data.

[ctculture.csv](https://github.com/user-attachments/files/16818345/ctculture.csv)

**Comment by AJLinn** [comment]
Here's my working file so you can see what I'm doing so far:
https://docs.google.com/spreadsheets/d/1qwNUsFb3c785Zk9CVXRjPrkXeKGt3dBCgX9DhUrU4OU/edit?usp=sharing

**Comment by dustymc** [comment]
The format should work - just don't touch CULTURE, I'll use it to get back to the data. The comments are being ignored in CSV download, so those don't seem to count as 'touch.'

**Comment by AJLinn** [comment]
> You might consider losing the commas in values, they seem to find ways to break things (mostly list operations).

I've looked in a couple of other online databases and here are some other models.

National Museum of the American Indian - seems to be an inconsistent use of either brackets or parenthesis:
<img width="1067" alt="Screenshot 2024-09-19 at 6 23 07 AM" src="https://github.com/user-attachments/assets/ec6b0951-7217-4dc3-999a-260414d1c7e2">

University of British Columbia Museum of Anthropology
<img width="507" alt="Screenshot 2024-09-19 at 7 32 39 PM" src="https://github.com/user-attachments/assets/ed96efd7-2938-48a1-b3b6-c7ec2b61f9c8">

Phoebe A. Hearst Museum of Anthropology
<img width="286" alt="Screenshot 2024-09-19 at 7 42 03 PM" src="https://github.com/user-attachments/assets/ad898e4d-34b9-44ff-b3ae-960a525f131f">

Looks like the most common way to handle this is to drop the inverted order that I'm so fond of for sorting! Guess I'll get over it.

**Comment by Jegelewicz** [comment]
@AJLinn just drop the comma and keep your sorting or replace the comma with something that (probably) doesn't break things like a pipe | OR use the metadata column in the code table:

culture | sort culture | metadata 
 -- | -- | -- 
Canadian Inuit | Indigneous people of the arctic region of Canada; see also http://en.wikipedia.org/wiki/Inuit | Inuit 
Copper Inuit | Indigenous people of northern Canada, in Nunavut's Kitikmeot Region and the Northwest Territories' Inuvik Region; see also http://en.wikipedia.org/wiki/Copper_Inuit | Inuit 
Greenlandic Inuit | Indigenous people of Greenland; see also http://en.wikipedia.org/wiki/Greenlandic_Inuit | Inuit 
Inuit | Indigenous people of the northern circumpolar region from eastern Siberia, across Alaska, Canada, and Greenland; see also http://en.wikipedia.org/wiki/Eskimo; formerly known as Eskimo | Inuit 

Then ask for sorting firts by metadata, then by culture.

**Comment by AJLinn** [comment]
At long last I've finished the first pass at totally re-writing the ctculture code table. For all those collections who use this table, 

PLEASE take a look to see that I've accurately represented those cultures in your region. I've tagged all in the Cultural Collections Working Group.

NOW is the time to add cultures that are needed by your collection!

@droberts49 @DellaCHall @brandon-s-thompson @wellerjes @sjshirar 

@mkoo @dustymc if you see anything structurally you'd like changed in this iteration please let me know.

**Comment by dustymc** [comment]
>Do dashes ( - ) work any better than commas?

Dashes aren't a "natural delimiter" to any language that we use so probably - but there are a bunch of similar-appearing unicode characters, so they're not much fun to type if that's a concern.

**Comment by AJLinn** [comment]
> I find the inverted sorting option so helpful; 

I totally agree and was physically pained by having to remove them for the programatic concerns raised by Dusty above. 

That said, if we have the new CT structure implemented for this one (as seen in the [life stage one here](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctlife_stage)) perhaps we can add in the "search terms column" those alternative names and group names (e.g., Inuit, Athabascan, etc). I'm not sure that work in the same way as the current inverted option that is visible while cataloging, but another possible option to consider?

**Comment by AJLinn** [comment]
> I thought https://github.com/ArctosDB/arctos/issues/8066#issuecomment-2364531123 had some good suggestions for helping us keep the sorting.

Agreed, but in the new table format there is no discreet metadata field. We could use the new "search terms" column to do essentially the same thing.

**Comment by dustymc** [comment]
>no discreet metadata field

That mechanism isn't going away, but it's not something I'd ever considered exposing for cataloging either. It's not part of any table, there's a whole 'nuther layer behind-ish the scenes.

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctculture

<img width="215" alt="Image" src="https://github.com/user-attachments/assets/236755dc-f184-48ad-b918-b5641b22276c" />

<img width="1232" alt="Image" src="https://github.com/user-attachments/assets/f11e381f-c418-472b-b9e6-e58db1901b89" />

An example of that being used is https://arctos.database.museum/info/ctDocumentation.cfm?table=ctchronostrat_stage_age - that powers eg  

<img width="235" alt="Image" src="https://github.com/user-attachments/assets/e2eb773e-11f3-482d-a70b-37d999eddde9" />

With development we could more formally say "Canadian Inuit is part of Inuit" in the same way we say eg "Aalenian is part of Mesozoic"

<img width="916" alt="Image" src="https://github.com/user-attachments/assets/a68898ad-e3ba-497b-9cf4-264804e22982" />

but that also (unless there's metadata and standards and such hiding somewhere) seems a bit overkill, I suspect some form of ``Inuit: Canadian Inuit`` sufficiently conveys the information, is impossible to avoid (would be overwhelming and probably controversial/impossible to keep up with eg stratigraphy), and is a huge simplification.

**Comment by AJLinn** [comment]
Agreed this should be visible in the pop-up list. Metadata is less valuable when one is making selections.

@dustymc Are colons preferred over pipes? Visually I don't have a preference.

Inuit: Canadian Inuit
Inuit: Copper Inuit
Inuit: Greenlandic Inuit

OR

Inuit | Canadian Inuit
Inuit | Copper Inuit
Inuit | Greenlandic Inuit

**Comment by AJLinn** [comment]
Culture WG also discussed adding non-Indigenous cultures to the CT. This would include ancient cultures (e.g., Ancient Roman is already there, though we could make that "Roman: Ancient" or "Roman | Ancient" to stay consistent with other previously comma-inverted forms) and contemporary.

How do folks more broadly feel about combining what might more accurately be labeled "nationalities" (e.g., Japanese, German, Icelandic, Norwegian) to the code table? I don't mind, though it has the chance to become unusably long if we add every single nationality in existence... We need to be able to accurately describe a culture of creation that has certain characteristics that describe elements of that work and we'd want to search for, and which might not be tied only to place of creation (e.g., a Japanese artist who moves to California but continues to create traditional Japanese art should be able to have culture of creation listed as Japanese, while the place of creation is California).

We also need some way to consistently list a culture that does not exist on the CT. I've been using "unknown" and then putting the name of the culture in the remark column, with some explanation. This can be searched for (if one knows to look for it there) using the full Catalog Record Attribute search option. Perhaps we should have an "Other | See remark" option?

Other ways to do this are invited for discussion!

**Comment by AJLinn** [comment]
Add https://native-land.ca/ to the definition to the code table page

**Comment by dustymc** [comment]
>Metadata is less valuable when one is making selections.

It shouldn't be, hoping two people somehow find identical meaning in a bare string often doesn't work out.

>colons preferred over pipes

I'm sure I've just been trained by the DWC folks, but in my brain pipes have become lists and so think I have a soft preference for the colons as clarifiers. Technically, I think they're equivalent and either will work.

>adding non-Indigenous cultures to the CT

I don't see much problem with that, as long as it somehow doesn't end up overlapping and arbitrary.

>more accurately be labeled "nationalities"

I suspect that split would immediately find that "overlapping and arbitrary" danger zone. If those are two clearly defined concepts then perhaps we need a second CT/term, if they're not (and I think they're not) then I think they make sense here. (And we **always** outgrow our names - hence the need for metadata/concepts!)

>every single nationality in existence

There's no reason to preemptively add anything, so let's just focus on the immediate needs for now. (And https://github.com/ArctosDB/BackBurner/issues/19 is somehow apparently happily working with a ridiculously long list - I suspect there's some relationship between list length and the "expertness" of the users, and this probably still aligns with quad.)

>culture of creation listed as Japanese, while the place of creation is California

Yes, clearly defined concepts are important! (And no problem, one is this attribute, the other is an event. And the attribute can carry metadata - just how Japanese that Californian can be is a bit subjective and will need it.)

>consistently list a culture that does not exist on the CT

Elaborate please - is there some reason not to simply add it? (There are almost certainly many of them, so maybe a Best Practices supplement to the CT and terms concepts would be valuable. ~ https://github.com/orgs/ArctosDB/discussions/6179)

>"Roman: Ancient"

No strong opinions, but that doesn't really fit in with the "General: Specific" form laid out by Inuit: Canadian Inuit. (I also don't know how to separate it from tourist trinkets bought in Rome - and probably made in China! - last week....)

**Comment by AJLinn** [comment]
[Updated CT](https://docs.google.com/spreadsheets/d/1qwNUsFb3c785Zk9CVXRjPrkXeKGt3dBCgX9DhUrU4OU/edit?usp=sharing) is ready to be reviewed and then sent to the code table admins for implementation.

**Comment by mkoo** [comment]
Above issues were discussed in Culture committee meeting yesterday.

I'm closing this issue -- all related comments and updates please go to #8989
